src/python/scripts/run_gerp.py

```diff
@@ -28,45 +28,81 @@
     $ python run_gerp.py --msa_file alignment.mfa --tree_file tree.nw --gerp_exe_dir path/to/gerp/
 
 """
-
 import os
 import subprocess
+import argparse
+import json
 
-import argschema
 
+def add_ce_to_json(ce_file: str, json_file: str) -> None:
+    """Enrich the json file with the constrained elements
 
-class InputSchema(argschema.ArgSchema):
-    """Calculates the Genomic Evolutionary Rate Profiling (GERP) of a multiple sequence alignment (MSA)."""
+    Args:
+        ce_file: file containing the constrained elements.
+        json_file: json file with genome coordinate level.
 
-    msa_file = argschema.fields.InputFile(
-        required=True, description="MSA file (MFA format)"
-    )
-    tree_file = argschema.fields.InputFile(
-        required=True, description="Tree file (Newick format). Must include every species in the MSA."
-    )
-    depth_threshold = argschema.fields.Float(
-        required=False,
-        description="Constrained elements' depth threshold for shallow columns, in substitutions "
-                    "per site. By default, 0.5."
-    )
-    gerp_exe_dir = argschema.fields.InputDir(
-        required=False,
-        description="Path where 'gerpcol' and 'gerpelem' binaries can be found. By default, resort to $PATH."
-    )
+    Returns:
+        None
+    """
+    with open(json_file, 'r') as json_file_handler:
+        align_set = json.load(json_file_handler)
 
+    with open(ce_file) as ce_file_handler:
+        constrained_elements = [ce.split() for ce in ce_file_handler if ce.rstrip() != ""]
 
-if __name__ == "__main__":
-    mod = argschema.ArgSchemaParser(schema_type=InputSchema)
+    # enrich json file with constrained elements info
+    constrained_elems = []
+    for ce in constrained_elements:
+        constrained_elem = {}
+        constrained_elem["start"] = ce[0]
+        constrained_elem["end"] = ce[1]
+        constrained_elem["length"] = ce[2]
+        constrained_elem["score"] = ce[3]
+        constrained_elem["p-val"] = ce[4]
```

Comment:
I'd advocate converting the constrained element field values to their respective numeric types here, so that they'll be stored as those types in the output JSON file.

I can see this is not a simple change though, because when I tried to implement it as follows...
```suggestion
        constrained_elem = {}
        constrained_elem["start"] = int(ce[0])
        constrained_elem["end"] = int(ce[1])
        constrained_elem["length"] = int(ce[2])
        constrained_elem["score"] = float(ce[3])
        constrained_elem["p-val"] = float(ce[4])
```
...I encountered the `mypy` error:
```
error: Incompatible types in assignment (expression has type "float", target has type "int")
```
It looks like this is happening because `mypy` infers the dictionary value type to be `int` based on the types of the `start`, `end` and `length` fields.

One workaround (requiring us to `import typing`) would be to add a variable annotation to the `constrained_elem` dictionary, to explicitly say that the dictionary value could be either an `int` or `float`:
```suggestion
        constrained_elem: typing.Dict[str, typing.Union[float, int]] = {}
        constrained_elem["start"] = int(ce[0])
        constrained_elem["end"] = int(ce[1])
        constrained_elem["length"] = int(ce[2])
        constrained_elem["score"] = float(ce[3])
        constrained_elem["p-val"] = float(ce[4])
```

A second approach would be to set all the dictionary values in a single statement, so that `mypy` correctly infers the dictionary value type:
```suggestion
        constrained_elem = {
            "start": int(ce[0]),
            "end": int(ce[1]),
            "length": int(ce[2]),
            "score": float(ce[3]),
            "p-val": float(ce[4])
        }
```